### DISCUSSION TITLE
Filter Connected Records in Grid View

### DISCUSSION THREAD

**Discussion by SteveRobinson-FIShR** [cortexrd/Knack-Toolkit-Library#427] [Q&A]
Hi All

I hope someone may be able to assist me with the following.  I have searched the forums and not seen a solution as yet.

I have three tables in my app customers, accounts and projects.  In the live app I am trying to display a grid of customers that allows for inline edit and shows selected connected fields from the other tables (accounts and projects - with a record status of active only).  At present I can set up the grid and show the connected records but cannot apply a filter to the results.  Is there a keyword that can be used to do this I have tried linked filters without success (I believe the filter only works on tables showing the same initial data set).

I can see that the ability to able to filter connected records has so many use cases and I am sure someone must have done this previously.

Thanks in Advance

Steve

**Comment by Greenlight88** [comment]
Hi Steve,

If I understand you correctly, I think you just need to write a text formulae that pulls in the data you want to filter from the other tables via the connection then apply a filter to that  eg ={Customers Accounts.Status}  or similar

more of a Knack problem than a KTL problem, good luck with your app
